tests/integration/scripts/load_file.c

```diff
@@ -0,0 +1,17 @@
+#include <stdio.h>
+#include <unistd.h>
+#include <stdlib.h>
+
+int main() {
+    // Program to execute
+    char *program = "/bin/ls";
```

Comment:
Why do we need a .c? Wouldn't feasible via a shell script?

Reply:
Its possible to trigger it without a c file, like that:

Tracee:
```
sudo ./dist/tracee -s comm=bash -e security_bprm_check.data.pathname=/usr/bin/ls -e shared_object_loaded.data.pathname=*libc.so.6 -e security_file_mprotect.data.pathname=*bash
```
Trigger:
```
bash -c 'exec /bin/ls -l'
```

But using `comm=bash`, Tracee will capture some events from OS. If we use a .c and have a binary called `load_file` we are able to filter it out using
```
sudo ./dist/tracee -s comm=load_file -e security_bprm_check.data.pathname=/usr/bin/ls -e shared_object_loaded.data.pathname=*libc.so.6 -e security_file_mprotect.data.pathname=*load_file
```

In this way, we capture events only from `load_file`, ensuring that our trigger generates them.

Reply:
What about doing this?

https://github.com/aquasecurity/tracee/pull/4414/files#diff-ae44cc6ba58bcc55063526ebf3fb632bf1f5ec14fba3051ea09ff2a5014605c3R10-R11

Reply:
Tks for the ref... I've simple create standalone script file with a ls command, it solved the problem... I was overthink it ahha